### PR TITLE
[site] Fix font size and color of texts and titles in main styles

### PR DESCRIPTION
**Pull request by ToyTox**
## Description
Fix font size and color of texts and titles in main styles.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Fix font size and color of texts and titles in main styles.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/css/docs.scss [hunk 1]
```diff
@@ -693,13 +698,15 @@ a.comparison-table__module-proprietaryOkmeter::after {
     font-weight: normal;
     font-style: normal;
     font-stretch: normal;
-    //color: #00122c;
-    color: $color-text-secondary;
+    color: #00122c;
+    //color: $color-text-secondary;
```
Comment (z9r5) [inline docs/site/css/docs.scss:702]:
```suggestion
```

File: docs/site/css/components/common/_variables.scss [hunk 2]
```diff
@@ -53,8 +53,8 @@ $font-size-lead: 54px;
 $font-size-h1: 36px;
 $font-size-h2: 28px;
 $font-size-h3: 22px;
-//$font-size-base: 14px;
-$font-size-base: 16px;
+$font-size-base: 14px;
+//$font-size-base: 16px;
```
Comment (z9r5) [inline docs/site/css/components/common/_variables.scss:57]:
```suggestion
```

**Review by z9r5 — APPROVED** [review]

File: docs/site/css/docs.scss [hunk 3]
```diff
@@ -708,7 +714,8 @@ a.comparison-table__module-proprietaryOkmeter::after {
 }
 
 .docs ul, .docs ol {
-    font-size: 15px;
+    //font-size: 15px;
```
Comment (z9r5) [inline docs/site/css/docs.scss:717]:
```suggestion
```

File: docs/site/css/docs.scss [hunk 4]
```diff
@@ -979,7 +985,8 @@ a.comparison-table__module-proprietaryOkmeter::after {
     font-stretch: normal;
     line-height: normal;
     letter-spacing: -0.3px;
-    color: #0066FF;
+    //color: #0066FF;
```
Comment (z9r5) [inline docs/site/css/docs.scss:988]:
```suggestion
```

**Pull request merged**